packages/svelte/src/reactivity/date.js

```diff
@@ -32,6 +32,14 @@ export class SvelteDate extends Date {
 			if (method.startsWith('get') || method.startsWith('to')) {
 				// @ts-ignore
 				proto[method] = function (...args) {
+					// If we have no params, then don't cache or create a derived
```

Comment:
comment says the opposite of the code

```suggestion
					// don't memoize if there are arguments
```